Update-Recover-Server (#2383)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeServer/high-availability/disaster-recovery/recover-exchange-servers.md

   4. Find the **msExchInstallPath** attribute. This attribute stores the current installation path.

- If you do not have the installation media for the Cumulative Update (CU) version that was installed on the server to be recovered, you can recover a server using the latest available Cumulative Update. Only the last two CUs are available for download. For more information, see [Updates for Exchange Server](../../new-features/updates.md). 

> Once the upgrade is successful, AdminDisplayVersion in EMS or msExchVersion attribute on recovered server will show old build number and this is a cosmetic in nature. We can either run setup /m:upgrade /IAcceptEchangeServerLicenseTerms  or wait for next Cumulative Update release and perform the upgrade which will correct this.

Reviewer: ```suggestion
Once the upgrade is successful, AdminDisplayVersion in EMS or the msExchVersion attribute on the recovered server will show the old build number and this is cosmetic in nature. You can either run setup /m:upgrade /IAcceptEchangeServerLicenseTerms or wait for the next Cumulative Update release and perform the upgrade which will correct this.
```

Author: Yes there was some mistake in the statement. Thanks for correcting and it seems good.

Reviewer: @sujnai - No problem. Could you please merge these changes? Thanks.

Author: I do not get option to merge it. Seems no write access for me.
